components/application-operator/charts/application/templates/authorization-policy.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if and (eq .Values.global.strictMode "enabled") (not .Values.global.centralApplicationConnectivityValidatorEnabled) }}
+{{- if and (.Values.global.disableLegacyConnectivity) (eq .Values.global.strictMode "enabled") (not .Values.global.centralApplicationConnectivityValidatorEnabled) }}
```

Comment:
Actually you could leave it as it is to minimise merge conflicts with https://github.com/kyma-project/kyma/pull/13296.

Reply:
Both PR will just delete the file. Conflict should be easily resolved.